src/Microsoft.AspNet.Mvc.ModelBinding/Binders/FormFileModelBinder.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Mvc;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class FormFileModelBinder : IModelBinder
+    {
+        public async Task<bool> BindModelAsync([NotNull] ModelBindingContext bindingContext)
+        {
+            if (bindingContext.ModelType == typeof(IFormFile))
+            {
+                var postedFiles = await GetFormFiles(bindingContext);
+
+                if (!postedFiles.Any())
+                {
+                    return false;
```

Comment:
Remove the false check. Instead do `postedFiles.FirstOrDefault();` and return true here all the time.